vip-helpers/class-wpcomvip-restrictions.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Automattic\VIP\Utils;
+
+class WPComVIP_Restrictions {
```

Comment:
Drive-by comment after-the-fact - since this class is already namespaced, we can drop the `WPComVIP_` prefix.

Reply:
`wpcomvip` relates to the user :-)